packages/components/src/combobox-control/index.js

```diff
@@ -279,6 +280,9 @@ function ComboboxControl( {
 							onHover={ setSelectedSuggestion }
 							onSelect={ onSuggestionSelected }
 							scrollIntoView
+							renderSuggestion={ ( { suggestion } ) =>
+								renderOption?.( { option: suggestion } )
+							}
```

Comment:
I believe this is breaking the default case where no `renderOption` prop is passed. In concrete, `renderSuggestion` is always a function regardless of the value of `renderOption`, which means that `if ( typeof renderSuggestion === 'function' ) {` (located at line 128 in packages/components/src/form-token-field/suggestions-list.tsx) will be always true.

In the end, the default case displays an empty string for each option:
![image](https://user-images.githubusercontent.com/3418513/181057009-15847f36-9829-4197-a22b-730e8e692479.png)

Reply:
Oh yeah, nice catch! Thanks @rcanepa

Reply:
Pushed a fix.